src/converters/utils/preparePullRequestStats.ts

```
@@ -34,6 +34,30 @@ export const preparePullRequestStats = (collection: Collection) => {
       collection.timeToMerge?.map((el) => el / 60),
       mergeIntervals
     ),
+    sizes: Object.entries(collection?.sizes || {}).reduce(
+      (acc, el) => ({
+        ...acc,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)